File: Bab 8/8.6.2.3 Count Character A in Array of String.java
import java.util.Scanner;

public class Main{

	public static void main(String[] args) {
		
		String[] string = new String[10];
		int penghitung = 0, huruf = 0, jumlah = 0;

		Scanner sc = new Scanner(System.in);

		for (penghitung = 0; penghitung < 10; penghitung++) {

			System.out.println("Masukan angka:\n");

			try{

				string[penghitung] = sc.next();

			}catch(Exception e){

			}
		}

		for (penghitung = 0; penghitung < 10; penghitung++)
		{
			for (huruf = 0; huruf < string[penghitung].length(); huruf++)
			{
				if ((string[penghitung].chartAt(huruf) == 'A') || (string[penghitung].chartAt(huruf) == 'a'))
				{
					jumlah++;
				}
			}
		}

		System.out.println(jumlah);
	}
}

File: Bab 8/8.6.2.3 Count Character A in Array of String.c
#include <stdio.h>
#include <string.h>

int main()
{
	
	char string[10][50];
	int penghitung, huruf, jumlah;

	for (penghitung = 0; penghitung < 10; penghitung++)
	{
		
		printf("Masukan kata:\n");
		scanf("%s", &string[penghitung]);
		printf("\n");

	}

	jumlah = 0;

	for (penghitung = 0; penghitung < 10; penghitung++)
	{
		for (huruf = 0; huruf < strlen(string[penghitung]); huruf++)
		{
			if ((string[penghitung][huruf] == 'A') || (string[penghitung][huruf] == 'a'))
			{
				jumlah++;
			}
		}
	}

	printf("%d\n", jumlah);

	return 0;
}